bindings/python/benchmark/__init__.py

```diff
@@ -60,3 +60,8 @@ def _run_benchmarks(argv):
 
 def main(argv=None):
   return app.run(_run_benchmarks, argv=argv, flags_parser=_flags_parser)
+
+
+# Methods for use with custom main function.
+initialize = _benchmark.Initialize
+run_benchmarks = _benchmark.RunSpecifiedBenchmarks
```

Comment:
`run_benchmarks = _run_benchmarks` ? Then you get the common usage error check on argv.

Reply:
This is for use with custom main functions, so there may be `argv` entries, or it might not be convenient to pass in `argv` at all.